mula/tests/integration/test_normalizer_scheduler.py

```diff
@@ -131,98 +195,193 @@ def test_populate_normalizer_queue_error_mimetype(self, mock_get_normalizers, mo
         task = functions.create_task(p_item)
         self.mock_ctx.task_store.create_task(task)
 
-        latest_raw_data = models.RawDataReceivedEvent(
-            raw_data=RawDataFactory(
-                boefje_meta=BoefjeMetaFactory(
-                    id=p_item.id.hex,
-                    boefje=boefje,
-                    input_ooi=ooi.primary_key,
+        boefje_meta = BoefjeMetaFactory(
+            id=p_item.id.hex,
+            boefje=boefje,
+            input_ooi=ooi.primary_key,
+        )
+
+        # Mocks
+        mock_get_latest_raw_data.side_effect = [
+            models.RawDataReceivedEvent(
+                raw_data=RawDataFactory(
+                    boefje_meta=boefje_meta,
+                    mime_types=[{"value": "text/plain"}],
                 ),
-                mime_types=[{"value": "error/boefje"}, {"value": "text/xml"}],
+                organization=self.organisation.name,
+                created_at=datetime.datetime.now(),
             ),
-            organization=self.organisation.name,
-            created_at=datetime.datetime.now(),
-        )
+            None,
+        ]
 
-        mock_get_latest_raw_data.side_effect = [latest_raw_data, None]
-        mock_get_normalizers.return_value = []
+        mock_get_normalizers_for_mime_type.return_value = []
+        mock_is_task_allowed_to_run.return_value = False
+        mock_is_task_running.return_value = False
```

Comment:
Since these return values are simple, you could consider putting them in the `patch` decorator instead. Although it might look a bit messy when doing so depending on the number of mocks and return value assignments. I'll leave it up to you!